core/cli/models.go

```diff
@@ -24,7 +25,8 @@ type ModelsList struct {
 }
 
 type ModelsInstall struct {
-	ModelArgs []string `arg:"" optional:"" name:"models" help:"Model configuration URLs to load"`
+	EnablePredownloadScan bool     `env:"LOCALAI_ENFORCE_PREDOWNLOAD_SCAN" help:"Runs the best-effort security scanner before downloading any files." default:"true"`
```

Comment:
ditto here

Reply:
ha, I even said I was going to fix it in both places and managed to forget in under 5 minutes 😆 . Fixed now.